charts/apisix/templates/configmap.yaml

```diff
@@ -45,7 +45,9 @@ data:
     {{- end }}
     {{- else }}
     apisix:    # universal configurations
+      {{- if not (eq .Values.deployment.mode "control_plane") }}
```

Comment:
Should also check this for SSL ports.

Reply:
Don't need , I have communicated with spacewander